manifests/config.pp

```diff
@@ -70,5 +97,25 @@
       notify  => Class['confluence::service'],
     }
 
+    file { "${confluence::webappdir}/confluence/WEB-INF/web.xml":
+      source  => 'puppet:///modules/confluence/web.xml',
+      mode    => '0600',
+      owner   => $::confluence::user,
+      group   => $::confluence::group,
+      require => Class['confluence::install'],
+      notify  => Class['confluence::service'],
+    }
+
+  }
+  # if JDBC was configured along with the license key and server_id, skip some server setup steps.
+  if $confluence::tomcat_jdbc_settings and $confluence::license and $confluence::server_id {
+    file { "${confluence::homedir}/confluence.cfg.xml":
```

Comment:
It's is an empty map by default.

I'm a little confused by the way this module is structured relative to what I've seen before (where all defaults are defined in params.pp and init.pp references them).